### PR TITLE
Adding action for testing conversion of each Cloud-Hosted-Guide

### PR DESCRIPTION
**Pull request by jakub-pomykala**
The original action has also been edited to support this.

### CODE REVIEW

File: .github/workflows/GuideConverterAction.yml [hunk 1]
```diff
@@ -150,7 +150,25 @@ jobs:
...
-            body: "Updated by github actions, triggered by the ${{ github.event.inputs.guide_name }} repo."
+            body: "Updated by github actions, triggered by the ${{ github.event.inputs.guide_name }} repo.   ![Testing-Conversion](https://github.com/jakub-pomykala/quick-labs/workflows/Testing-Conversion/badge.svg)"
             branch: "cloud-hosted-${{ github.event.inputs.guide_name }}/staging"
             base: "staging" 
-        name: Create Pull Request
\ No newline at end of file
+        name: Create Pull Request
+
+
+  post:
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yml:159]: Can you give this a descriptive name and remove the unnecessary comments please.

**Review by austin0 — CHANGES_REQUESTED** [review]
Small change

File: .github/workflows/GuideConverterAction.yml (hunk 1 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yml:159]: 👍🏻

Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yml]: Done

**Review by austin0 — APPROVED** [review]
LGTM

**Pull request merged**